lib/ExpressRedisCache/route.js

```diff
@@ -167,7 +167,7 @@ module.exports = (function () {
 
           if ( cache.length &&  cache[0].body != null ) {
             res.contentType(cache[0].type || "text/html");
-            res.status(cache[0].status);
+            res.status(cache[0].status || 200);
```

Comment:
If you do not know the correct status, it is better to drop the cache.

Reply:
valid suggestion... If only somebody maintained this lib at all.